### PR TITLE
code-generator: add ut in ./staging/src/k8s.io/code-generator/generators

### PR DESCRIPTION
**Pull request by tukwila**
What type of PR is this?

/kind cleanup

What this PR does / why we need it:

[UT]Increase ut for code-generator
https://github.com/kubernetes/kubernetes/issues/119289
k8s.io/code-generator/cmd/conversion-gen/generators coverage: 0.5% of statements

Which issue(s) this PR fixes:

Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?

NONE
Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

NONE

### CODE REVIEW

**Comment by tukwila** [comment]
cc @aojea

**Comment by jiahuif** [comment]
/assign @sttts
Could you take a look? Thank you.
/triage accepted

File: staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion_test.go [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+		"two same comments": {
+			comments: []string{
+				"+k8s:conversion-gen=k8s.io/kubernetes/runtime.Object",
+				"+k8s:conversion-gen=k8s.io/kubernetes/runtime.Object",
+			},
+			expect: []string{
+				"k8s.io/kubernetes/runtime.Object",
+				"k8s.io/kubernetes/runtime.Object",
+			},
+		},
```
Comment (sttts) [inline staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion_test.go:60]: looking through the function, there are many more things to test. Non-tag lines for example. Or other kind of tags.

Comment (tukwila) [reply staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion_test.go:88]: yes, add other tags ut.

**Pull request closed without merging**